nixos/modules/services/x11/urserver.nix

```diff
@@ -0,0 +1,72 @@
+# urserver service
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.urserver;
+in {
+
+  options.services.urserver = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable Unified Remote server.
+      '';
+    };
+
+    usedTCPPorts = mkOption {
+      type = types.listOf types.int;
+      default = [ 9510 9512 ];
+      description = ''
+        TCP ports that are used by urserver.
+        By default port 9510 is used for the web server.
+        By default port 9512 is used for WiFi connections.
+      '';
+    };
+
+    usedUDPPorts = mkOption {
+      type = types.listOf types.int;
+      default = [ 9511 9512 ];
+      description = ''
+        UDP ports that are used by urserver.
+        By default port 9511 is used for automatic server discovery.
+        By default port 9512 is used for fast connections.
+      '';
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.urserver ];
+
+    systemd.services.urserver = {
+      description = ''
+        Server for Unified Remote: The one-and-only remote for your computer.
+      '';
+      wantedBy = [ "graphical.target" ];
```

Comment:
This makes me wonder if this package should be run as a regular desktop user, which might mean it should be a `systemd` `user` service... :thinking:

Reply:
See above.

Thanks for the quick reviews - I will have to pick it up again tomorrow.

Reply:
It now runs as the regular user after login to the desktop.